lib/server/handlers/sthGetDataHandler.js

```diff
@@ -200,16 +206,21 @@ function getAggregatedData(request, reply) {
     );
     sthDatabase.getCollection(query, sthOptions, function(err, collection) {
         if (err) {
-            // The collection does not exist, reply with en empty response
-            sthLogger.warn(
-                request.sth.context,
-                'Error %j when getting the aggregated data collection for retrieval' +
-                    // Overly long line.
-                    ' using query %j and options %j',
-                err,
-                query,
-                sthOptions
-            );
+            if(err.name === 'MongoNetworkError' && err.message.includes('failed to connect to server')){
```

Comment:
Aligned with https://github.com/telefonicaid/fiware-sth-comet/pull/583/files#r813167242, maybe just:

```suggestion
            if(err.name === 'Not connected')){
```